src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -55,14 +58,23 @@
     @Nullable
     private EventTypeOptions options;
 
-    public EventType() {}
+    private Set<String> writeScopes;
+
+    private Set<String> readScopes;
+
+    public EventType() {
+        this.validationStrategies = Lists.newArrayList();
+        this.enrichmentStrategies = Lists.newArrayList();
+        this.partitionStrategy = PartitionStrategy.RANDOM_STRATEGY;
+        this.writeScopes = Collections.emptySet();
+        this.readScopes = Collections.emptySet();
```

Comment:
For me it looks very strange that for initialising collections we use two different approaches within 5-line method :)
(Guava and java.util.Collections) 
I would prefer to have homogenous approach (at least within the same method) doesn't really matter which one.